Source/WebCore/html/FileInputType.cpp

```diff
@@ -383,28 +384,30 @@ void FileInputType::filesChosen(const Vector<FileChooserFileInfo>& paths, const
     if (!displayString.isEmpty())
         m_displayString = displayString;
 
-    if (m_directoryFileListCreator)
-        m_directoryFileListCreator->cancel();
+    if (RefPtr directoryFileListCreator = m_directoryFileListCreator)
```

Comment:
Note that Darin likes using short names for locally scoped variables like this. I would have probably used `creator` here to be more concise.